app/workers/daily_grouping_updater.rb

```diff
@@ -0,0 +1,27 @@
+class DailyGroupingUpdater
+  include Sidekiq::Worker
+
+  def initialize
+    @system_account = Watcher.find_by_email("kairos-robot@omadahealth.com")
+    PaperTrail.whodunnit = @system_account
+  end
+
+  def perform(app_name: nil)
+    raise "An app_name must be specified." unless app_name.present?
+
+    groupings = Grouping.app_name(app_name).where(state: :acknowledged).where("latest_wat_at <= ?", 15.days.ago)
+    groupings.each do |grouping|
+      grouping.resolve!
+
+      accept_tracker_story(grouping.pivotal_tracker_story_id) if grouping.pivotal_tracker_story_id.present?
+    end
+  end
+
+  private
+
+  def accept_tracker_story(story_id)
+    story = @system_account.tracker.story(story_id)
```

Comment:
Now that you mention it, it'd be good to probably do this in general. I'll make that change.